### PR TITLE
feat: use `cobra` library instead of `flag`

### PR DESCRIPTION
**Pull request by d3cryptofc**
```
$ ./hyprnotify --help

DBus Implementation of Freedesktop Notification spec for 'hyprctl notify'

Usage:
  hyprnotify [flags]

Flags:
  -h, --help       help for hyprnotify
  -s, --no-sound   disable sound, silent mode
```

I'm suggesting this PR for the following reasons:

- It has 'shorthand', a short version of the command, resulting in a clearer manual to read and avoiding the need to create several flags as aliases.
- In parallel I'm working on adding a flag called `--font-size` to define a fixed size for the font, and this library would make reading much more pleasant. (In fact, I already have the code ready and working, but I still need to organize the commits)

I know it's a boring situation but I ask you to think about it, I think it's worth it. Thank you!

### CODE REVIEW

**Comment by codelif** [comment]
dude you are on fire 🔥

**Comment by d3cryptofc** [comment]
> dude you are on fire 🔥

I'm tired of Mako and Dunst, when I found this project I almost died of happiness haha

**Comment by codelif** [comment]
> I know it's a boring situation but I ask you to think about it, I think it's worth it. Thank you!

No it's not a boring situation. Infact it might make #6 easier to implement.

**Comment by codelif** [comment]
>     * In parallel I'm working on adding a flag called `--font-size` to define a fixed size for the font, and this library would make reading much more pleasant. (In fact, I already have the code ready and working, but I still need to organize the commits)

gosh, hats off to the 10x dev. Made my day.

**Comment by d3cryptofc** [comment]
Is there anything missing from this PR?

**Comment by codelif** [comment]
see the review

**Comment by d3cryptofc** [comment]
OK. By the way, sorry, I forgot about the time zone, here in my country it has just fallen dusk.

File: cmd/hyprnotify/main.go [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+		Run: func(cmd *cobra.Command, args []string) {
+			internal.InitDBus(enableSound)
+		},
+	}
 
-	flag.Parse()
+	CmdFlags := Cmd.Flags()
 
-	internal.InitDBus(enableSound)
+	CmdFlags.BoolVarP(&enableSound, "no-sound", "s", false, "disable sound, silent mode")
```
[the remark this answers is not in the document]
Comment (d3cryptofc) [reply cmd/hyprnotify/main.go:21]: I just copied the original, how strange is that...

https://github.com/codelif/hyprnotify/blob/main/cmd/hyprnotify/main.go#L12-L14

**Comment by codelif** [comment]
LGTM!

**Pull request merged**